client-cli/src/test/java/org/eclipse/dataspaceconnector/identityhub/cli/VerifiableCredentialsCommandTest.java

```diff
@@ -71,6 +71,25 @@ void setUp() {
         cmd.setErr(new PrintWriter(err));
     }
 
+    @Test
+    void getSelfDescription() {
+        var selfDescription = MAPPER.createObjectNode();
+        selfDescription.put(FAKER.lorem().word(), FAKER.lorem().word());
+        // arrange
+        when(app.identityHubClient.getSelfDescription(app.hubUrl)).thenReturn(success(selfDescription));
+
+        // act
+        var exitCode = executeGetSelfDescription();
+        var outContent = out.toString();
+        var errContent = err.toString();
+
+        // assert
+        assertThat(exitCode).isZero();
+        assertThat(errContent).isEmpty();
+
+        assertThat(outContent).isEqualToIgnoringWhitespace(selfDescription.toString());
```

Comment:
why ignoring whitespace? If we expect the JSON to be seen by users, then things like multiline and final EOL are not ancillary, but part of the test scope.

Reply:
Updated